backend/transcribee_backend/auth.py

```diff
@@ -50,8 +50,8 @@ def validate_user_authorization(session: Session, authorization: str):
         raise HTTPException(status_code=401)
 
     try:
-        token_data = b64decode(token.encode()).decode()
```

Comment:
Why this?

Reply:
`b64decode` already internally encodes ascii strings, so this seems unnecessary?